e2e/browser/src/components/notifications/index.tsx

```diff
@@ -44,15 +158,15 @@ export default function Notifications() {
   }, []);
 
   useEffect(() => {
-    if (parentContainerUrl !== undefined) {
+    if (parentContainerUrl !== undefined && notificationGateway !== undefined) {
       setSocket(
         new WebsocketNotification(parentContainerUrl, {
           fetch: getDefaultSession().fetch,
-          gateway: "https://notification.inrupt.com",
+          gateway: notificationGateway,
```

Comment:
isn't this automatically discovered based on the pod host via the well-known?

Reply:
That's correct, I was hung up on an old bug that has been long resolved. Removed in 8c85c8bcffd2cab41de1c9848eb81fbe5ae97f26